### ISSUE TITLE
Flask-REST Marshmallow Two-way Nesting Schemas

### ISSUE THREAD

**Issue by froggylab** [di/flask-accept#10]
TL;DR : Is there a way to accept string Nested marshmallow fields ?

In order to avoid circular import in a flask/marshmallow project, it's possible to reference the Nested field by using its name (as described here https://marshmallow.readthedocs.io/en/latest/nesting.html#two-way-nesting) 

Unfortunately, flask_accept doesn't support it : 

`  File env/lib/python3.8/site-packages/flask_accepts/decorators/decorators.py", line 110, in decorator
    body = for_swagger(
  File "env/lib/python3.8/site-packages/flask_accepts/utils.py", line 63, in for_swagger
    fields = {
  File "env/lib/python3.8/site-packages/flask_accepts/utils.py", line 64, in <dictcomp>
    k: map_type(v, api, model_name, operation)
  File "env/lib/python3.8/site-packages/flask_accepts/utils.py", line 182, in map_type
    return type_map[value_type](val, api, model_name, operation)
  File "env/lib/python3.8/site-packages/flask_accepts/utils.py", line 19, in unpack_nested
    model_name = get_default_model_name(val.nested)
  File "env/lib/python3.8/site-packages/flask_accepts/utils.py", line 152, in get_default_model_name
    return "".join(schema.__name__.rsplit("Schema", 1))

AttributeError: 'str' object has no attribute '__name__'
`

Can you please add the possibility to support this configuration ?

**Comment by di** [comment]
Hi, could you provide a minimal example that reproduces this? I'm not very familiar with Marshmallow.
